docs/drill4net/drill4j-admin-installation.mdx

```diff
@@ -0,0 +1,6 @@
+---
+id: drill4j-admin-installation
+title: Drill4J Admin Installation
+---
+
+Drill4J Admin installation process is described [here](./../installation/drill-admin.mdx).
```

Comment:
`../installation/drill-admin.mdx ` provides information on setup of **older version** of Drill4J

I suggest you copy  both `docker-compose.yml` and `.env` that you're using to deploy Drill4J Admin right here. (since they contain up-to-date component versions)